extension/src/openvic-extension/classes/GUILabel.cpp

```diff
@@ -724,8 +693,7 @@ std::vector<GUILabel::line_t> GUILabel::wrap_lines(std::vector<line_t>& unwrappe
 							if (last_marker_pos != 0 || !current_line->segments.empty()) {
 								if (!new_segment_string.is_empty()) {
 									current_line->segments.emplace_back(string_segment_t {
-										std::move(new_segment_string), string_segment->colour, new_segment_width
-									});
+										std::move(new_segment_string), string_segment->colour, new_segment_width });
```

Comment:
Ah I thought the `{}` handling would break before the `}` too, I'm fairly sure the old formatting settings had something that did that?